src/Options.php

```diff
@@ -789,6 +814,7 @@ private function configureOptions(OptionsResolver $resolver): void
         $resolver->setAllowedTypes('in_app_exclude', 'array');
         $resolver->setAllowedTypes('project_root', ['null', 'string']);
         $resolver->setAllowedTypes('logger', 'string');
+        $resolver->setAllowedValues('error_logger', \Closure::fromCallable([$this, 'validateLoggerOption']));
```

Comment:
I think you should be able to use `setAllowedTypes('error_logger', LoggerInterface::class);` here

Reply:
Yes, this works. Thanks. Will update the PR